doc/Pacemaker_Explained/en-US/Ch-Resources.txt

```diff
@@ -519,7 +519,7 @@ but moderate. The minimum timeouts should never be below 10 seconds.
 Location to store the resource state in.
 </longdesc>
 <shortdesc lang="en">State file</shortdesc>
-<content type="string" default="/var/run//Dummy-{OCF_RESOURCE_INSTANCE}.state" />
+<content type="string" default="/var/run//Dummy-default.state" />
```

Comment:
It turns out this commit is the right thing to do. ocf-shellfuncs defaults OCF_RESOURCE_INSTANCE to "default" or "undef", so the rest of the change needs to be there instead. Submitted as https://github.com/ClusterLabs/resource-agents/pull/778

The pacemaker agents will still say "Dummy-default.state" etc. until resource-agents is updated, at which point they will say "Dummy-RESOURCE_ID.state" etc. if OCF_RESOURCE_INSTANCE is unspecified, and the actual value if it is specified.

Separately, I'm going to clean up these agents a bit ...